src/components/autocomplete-dropdown/index.tsx

```diff
@@ -52,19 +52,31 @@ export class AutocompleteDropdown extends React.Component<Properties, State> {
       currentFocusIndex: 0,
     };
 
-    this.performSearch = debouncePromise(this._performSearch, config.debounceRate);
+    this.performSearch = debouncePromise(
+      this._performSearch,
+      config.debounceRate
+    );
+    this.escFunction = this.escFunction.bind(this);
   }
 
-  _performSearch = async searchTerm => {
-    return this.props.findMatches(searchTerm);
+  escFunction= async (event) => {
+    if (event.key === 'Escape') {
```

Comment:
we have an enum `Key`, can you add the `escape` to it.
path: `src/lib/keyboard-search.ts`